File: php/testcode/benchmark_test_00698.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00698(BenchmarkRequest $req): BenchmarkResponse {
    $uploadPath = '/var/www/uploads/';
    $dest = $uploadPath . 'file.dat';
    extract($_FILES);
    return BenchmarkResponse::ok("dest=$dest");
}

File: rust/testcode/benchmark_test_00698.rs
fn db_get_note(note_id: &str) -> String {
    format!("private_note_content_for_{}", note_id)
}

pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let note_id = req.param("note_id");
    let note = db_get_note(&note_id);
    super::shared::BenchmarkResponse::ok(&note)
}
